src/Microsoft.VisualStudio.Services.Agent/HostContext.cs

```diff
@@ -17,11 +17,13 @@
 using System.Diagnostics;
 using System.Net.Http;
 using System.Diagnostics.Tracing;
-using Microsoft.TeamFoundation.DistributedTask.Logging;
 using System.Net.Http.Headers;
+using Agent.Sdk.SecretMasking;
 using Pipelines = Microsoft.TeamFoundation.DistributedTask.Pipelines;
 using Agent.Sdk.Util;
 using BuildXL.Cache.ContentStore.Interfaces.Tracing;
+using Microsoft.TeamFoundation.DistributedTask.Logging;
+using SecretMasker = Agent.Sdk.SecretMasking.SecretMasker;
```

Comment:
Discussed offline with Kirill - we should have a FF to enable the legacy path (similar to #4489)

e.g. in #4489 HostContext.cs:
            _secretMasker = AgentKnobs.MaskRefactoring.GetValue(this).AsBoolean()          
                ? new LoggedSecretMasker()
                : new LegacyLoggedSecretMasker(_legacySecretMasker);

Reply:
@merlynomsft updated